.github/workflows/cd-hl7v23.yml

```diff
@@ -0,0 +1,127 @@
+name: CD - Hl7v23
+
+on:
+  workflow_dispatch:
+    inputs:
+      bal_central_environment:
+        description: Ballerina Central Environment
+        type: choice
+        options:
+        - STAGE
+        - DEV
+        - PROD
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      JAVA_OPTS: -Xmx4G
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action@2201.6.0
```

Comment:
Can we have this as a variable? otherwise each time when the version updates, we have to update every other place in the workflow. Do the same change for the other workflow files as well

Reply:
We can't use variables in container names. It's a known issue - https://github.com/actions/runner/issues/2394